domain/notifier.go

```diff
@@ -1,12 +1,12 @@
 package domain
 
 type NotificationMessages struct {
-	ExpirationReminder   string `mapstructure:"expiration_reminder" default:"Your access {{.account_id}} to {{.resource_name}} with role {{.role}} will expire at {{.expiration_date}}. Extend the access if it's still needed"`
-	AppealApproved       string `mapstructure:"appeal_approved" default:"Your appeal to {{.resource_name}} with role {{.role}} has been approved"`
-	AppealRejected       string `mapstructure:"appeal_rejected" default:"Your appeal to {{.resource_name}} with role {{.role}} has been rejected"`
-	AccessRevoked        string `mapstructure:"access_revoked" default:"Your access to {{.resource_name}}} with role {{.role}} has been revoked"`
-	ApproverNotification string `mapstructure:"approver_notification" default:"You have an appeal created by {{.requestor}} requesting access to {{.resource_name}} with role {{.role}}. Appeal ID: {{.appeal_id}}"`
-	OthersAppealApproved string `mapstructure:"others_appeal_approved" default:"Your appeal to {{.resource_name}} with role {{.role}} created by {{.requestor}} has been approved"`
+	ExpirationReminder   string `mapstructure:"expiration_reminder" default:"[{\"type\":\"section\",\"text\":{\"type\":\"mrkdwn\",\"text\":\"Your access {{.account_id}} to {{.resource_name}} with role {{.role}} will expire at {{.expiration_date}}. Extend the access if it's still needed\"}}]"`
+	AppealApproved       string `mapstructure:"appeal_approved" default:"[{\"type\":\"section\",\"text\":{\"type\":\"mrkdwn\",\"text\":\"Your appeal to {{.resource_name}} with role {{.role}} has been approved\"}}]"`
+	AppealRejected       string `mapstructure:"appeal_rejected" default:"[{\"type\":\"section\",\"text\":{\"type\":\"mrkdwn\",\"text\":\"Your appeal to {{.resource_name}} with role {{.role}} has been rejected\"}}]"`
+	AccessRevoked        string `mapstructure:"access_revoked" default:"[{\"type\":\"section\",\"text\":{\"type\":\"mrkdwn\",\"text\":\"Your access to {{.resource_name}}} with role {{.role}} has been revoked\"}}]"`
+	ApproverNotification string `mapstructure:"approver_notification" default:"[{\"type\":\"section\",\"text\":{\"type\":\"mrkdwn\",\"text\":\"You have an appeal created by {{.requestor}} requesting access to {{.resource_name}} with role {{.role}}.\\n Appeal ID: {{.appeal_id}}\"}}]"`
+	OthersAppealApproved string `mapstructure:"others_appeal_approved" default:"[{\"type\":\"section\",\"text\":{\"type\":\"mrkdwn\",\"text\":\"Your appeal to {{.resource_name}} with role {{.role}} created by {{.requestor}} has been approved\"}}]"`
```

Comment:
I'm not getting good vibes(for lack of a better word) with how the default value is configured in here. 
@rahmatrhd @haveiss what's your opinion on this? Is this fine or do we have to think of using a constructer?

Reply:
for the default values, we can create .json files and programmatically set them as default values here.
but for actual customization in production I don't think there would be a better way to set the value since it'll be retrieved from env vars 🤔

Reply:
Agreed. Lets use .json files in code and set default. 

For deployment, it should be fine from using env vars, as the type is string. And as long as the user can maintain these strings while deploying, it should not be an issue to fetch from env vars.

Reply:
For default templates, created templates folder with all the json inside it and used it programmatically in code.